package.json

```
@@ -8,9 +8,12 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "build": "webpack --watch"
+    "build": "webpack --watch",
```

I recommend use the -d option so webpack knows it's in development mode.
```suggestion
     "build": "webpack -d --watch",
```